traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/callbacks/WishlistFetchCallback.java

```diff
@@ -0,0 +1,20 @@
+package com.guestlogix.travelercorekit.callbacks;
+
+import androidx.annotation.Nullable;
+import com.guestlogix.travelercorekit.models.WishlistResult;
+
+public interface WishlistFetchCallback {
+    /**
+     * Called when the results have successfully been fetched
+     */
+    void onWishlistFetchSuccess(WishlistResult result, int identifier);
+    void onWishlistFetchError(Error error);
+    @Nullable WishlistResult getPreviousResult(int identifier);
```

Comment:
Can we document the rest of these methods? :)

Docs are available for the iOS equivalent interface

Reply:
updated